src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ContentReviewModule extends WikiBasePageObject {
+
+    @FindBy(css = ".content-review-module")
+    private WebElement contentReviewModule;
+    @FindBy(css = ".content-review-module-submit")
+    private WebElement submitForReviewLink;
+
+    public ContentReviewModule() {
+        super();
+    }
+
+    public boolean isModuleVisible(){
+        try {
+            wait.forElementVisible(contentReviewModule, 5, 1);
+            return true;
+        }catch (TimeoutException e){
+            return false;
+        }
+    }
+
+    public boolean isSubmitLinkVisible() {
+        boolean isLinkVisible = false;
+        try {
+            wait.forElementVisible(submitForReviewLink, 3, 1);
+            isLinkVisible = true;
+        } catch(TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)